wrapanapi/msazure.py

```diff
@@ -300,6 +305,17 @@ def list_subscriptions(self):
                 self.subscription_client.subscriptions.list() if
                 s.state == SubscriptionState.enabled]
 
+    def list_storage_accounts_by_resource_group(self, resource_group):
+        """List Azure Storage accounts on current subscription by resource group"""
+        return [s.name for s in self.storage_client.storage_accounts.list_by_resource_group(
+                resource_group)]
+
+    def get_storage_account_key(self, storage_account_name, resource_group):
```

Comment:
Why not make this method plural and return both keys, allowing the caller to pick which one they want (or to test both)

Reply:
keys are totally equal. the reason why Azure provides 2 keys is that in case you'll need regenerate any compromised key - just switch to another